cmd/node-agent/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/coreos/bootkube/pkg/node"
+	"github.com/coreos/go-systemd/dbus"
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/client/cache"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_3"
+	"k8s.io/kubernetes/pkg/client/restclient"
+	"k8s.io/kubernetes/pkg/controller/framework"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/wait"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+const myPodNameEnvName = "MY_POD_NAME"
+
+func main() {
+	myPodName := os.Getenv(myPodNameEnvName)
+	if myPodName == "" {
+		glog.Fatalf("%s env var not set.", myPodNameEnvName)
+	}
+
+	client := newAPIClient()
+	nodename, err := getNodeName(client, myPodName)
```

Comment:
Can you add a TODO note here to switch to using nodename directly once we're on v1.4